### PR TITLE
updated content for voting resource page

### PR DESCRIPTION
**Pull request by ilodidoj**
[Link to issue.](https://github.com/usdoj-crt/crt-portal-management/issues/1387)

## What does this change?
Voting content updated according to the updated source of truth.

### Author

+ [x] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.).
+ [x] Check for, document, and establish a testing plan for any behavior that may vary across environments or is otherwise difficult to test.
+ [x] Check for [accessibility](https://github.com/usdoj-crt/crt-portal/blob/develop/docs/a11y_plan.md).
+ [x] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [x] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.).
+ [x] Check for any behavior that may vary across environments or is difficult to test, and ensure that it is well-understood, documented, and that there is a testing plan in place.
+ [x] Re-check for [accessibility](https://github.com/usdoj-crt/crt-portal/blob/develop/docs/a11y_plan.md).
+ [x] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

## Notes for reviewer:

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

### CODE REVIEW

File: crt_portal/cts_forms/templates/vot_resources.html [hunk 1]
```diff
@@ -46,55 +47,67 @@ <h1 id="get-help">
...
+
+            <li>{% trans "After alerting local law enforcement to such emergencies by calling"%}
+            <a class="external-link--blue" href="tel:911">
+            {% trans "911" %}
+            </a>,
+            {% trans "the public should contact the Department of Justice" %} </li>
+
+            <li>{% trans "Election-crime complaints should be directed to your local" %}
+            <a class="external-link--blue" href="https://www.justice.gov/usao/find-your-united-states-attorney">U.S. Attorney's Offices </a>. {% trans "or the local" %}
+            <a class="external-link--blue" href="https://www.fbi.gov/contact-us">FBI office</a>
```
Comment (billyhunt) [inline crt_portal/cts_forms/templates/vot_resources.html:86]: Should "FBI office" be translated?

File: crt_portal/cts_forms/templates/vot_resources.html [hunk 2]
```diff
@@ -46,55 +47,67 @@ <h1 id="get-help">
...
-            </li>
-            <li>{% trans "Do you have a criminal conviction? Wondering if you can vote in your state?<br>Download the Justice Department's new " %}<a class="link--blue" href="https://www.justice.gov/voting/file/1507306/download">Guide to State Voting Rights that Apply After Criminal Conviction</a>.</li>
+            <li>{% trans "Find additional information about voting rights laws at " %}<a class="link--blue" href="https://justice.gov/crt/voting-section">www.justice.gov/crt/voting-section</a>.</li>
+
             <li>{% trans "Learn about federal laws that protect voters with disabilities at " %}<a class="link--blue" href="https://ada.gov">ada.gov</a>.</li>
-            <li>{% trans "Find additional information about voting rights at " %}<a class="link--blue" href="https://justice.gov">justice.gov</a>.</li>
+
+
+            <li>{% trans "Download the Justice Department's new " %}<a class="link--blue" href="https://www.justice.gov/file/1517936/download">Know Your Voting Rights</a> guide.
```
Comment (billyhunt) [inline crt_portal/cts_forms/templates/vot_resources.html:109]: should "Know your voting rights" be translated?

**Review by billyhunt — COMMENTED** [review]
Just a couple of potential translations.  Everything else looks good!

File: crt_portal/cts_forms/templates/vot_resources.html (hunk 2 above)
Comment (ilodidoj) [reply crt_portal/cts_forms/templates/vot_resources.html:109]: i think so.

File: crt_portal/cts_forms/templates/vot_resources.html (hunk 1 above)
Comment (ilodidoj) [reply crt_portal/cts_forms/templates/vot_resources.html:86]: yes

**Review by billyhunt — APPROVED** [review]
Looks good to me!

**Pull request merged**